code/web/src/modules/user/Profile.js

```diff
@@ -24,23 +26,67 @@ const Profile = (props) => (
     </Helmet>
 
     {/* Top title bar */}
-    <Grid style={{ backgroundColor: grey }}>
-      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+    <Grid style={{ backgroundColor: grey }} justifyRight={true}>
+      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '20vw', marginRight: '24%' }}>
         <H3 font="secondary">My profile</H3>
       </GridCell>
+      
+      <GridCell style={{ padding: '2em', textAlign: 'center', maxWidth: '9vw' }}>
+        <Button theme="secondary" onClick={props.logout} style={{ marginLeft: '1em' }}>Logout</Button>
+      </GridCell>
     </Grid>
 
     <Grid>
       <GridCell style={{ padding: '2em', textAlign: 'center' }}>
-        <H4 style={{ marginBottom: '0.5em' }}>{props.user.details.name}</H4>
+        <img
+          src={`${APP_URL}//images/stock/men/4.jpg`}
+          style={{ 
+            marginBottom: '2em', 
+            height: '24em', 
+            boxShadow: level1, 
+            borderRadius: '10px'
+          }}
+        /> 
+        <H4>{props.user.details.name}</H4>
+      </GridCell>
+
+      <GridCell style={{ padding: '2em', textAlign: 'center' }}>
+        <H4 font='secondary' style={{ marginBottom: '1.5em' }}>Account Details</H4>
 
+        <H6 style={{ marginBottom: '0.5em' }}>Account email:</H6>
         <p style={{ color: grey2, marginBottom: '2em' }}>{props.user.details.email}</p>
 
-        <Link to={userRoutes.subscriptions.path}>
-          <Button theme="primary">Subscriptions</Button>
-        </Link>
+        <H6 style={{ marginBottom: '0.5em' }}>Shipping Address:</H6>
```

Comment:
We need to talk about the way that shipping address will be processed on the backend today. From a Back end perspective, it might make sense to update this differently or create a different path for adding and then updating a shipping address

Reply:
To echo Sienna, we currently have created a table for shipping_address with string columns for street, city, state, and zip. We thought that would make life easier for frontend than adding an address text column to the users table.